common/config/presets/windows.toml

```
@@ -122,144 +122,11 @@ base_node_query_timeout = 120
 
 # Select the network to connect to. Valid options are:
 #   mainnet - the "real" Tari network (default)
-#   rincewind - the Tari test net
-network = "rincewind"
+#   Ridcully - the Tari test net
+network = "Ridcully"
```

```suggestion
network = "ridcully"
```
Not sure if this makes a difference, but probably worth keeping it the same